boto/s3/keyfile.py

```diff
@@ -84,9 +84,17 @@ def seek(self, pos, whence=os.SEEK_SET):
 
     self.location = pos
 
-  def read(self, size):
-    self.location += size
-    return self.key.read(size)
+  def read(self,*args):
```

Comment:
as I understand it would be better to use named parameter here instead of vararg. 

```
def read(self, size=-1):
```

2) Could you please document touched method

```
Read at most size bytes from the file (less if the read hits EOF before obtaining size bytes).

:param size: Size of the content to read in bytes. If the size argument is negative or omitted, read all data until EOF is reached. 
:type size: int
:return: .... `None` is returned when EOF is encountered.
:rtype: ?
```